powerauth-java-cmd-lib/src/main/java/com/wultra/security/powerauth/lib/cmd/util/TemporaryKeyUtil.java

```diff
@@ -124,6 +124,13 @@ private static byte[] getSecretKey(StepContext<? extends BaseStepData, ?> stepCo
         } else if (scope == EncryptorScope.ACTIVATION_SCOPE) {
             final byte[] appSecretBytes = Base64.getDecoder().decode(appSecret);
             final SecretKey transportMasterKey = model.getResultStatus().getTransportMasterKeyObject();
+            if (transportMasterKey == null) {
+                stepContext.getStepLogger().writeError(
+                        stepContext.getStep().id() + "-failed",
+                        "Get Secret Key Failed",
```

Comment:
Let's change the message to `Get Transport Master Key Failed`, there are many secret keys.